app/index.js

```diff
@@ -180,5 +195,23 @@ module.exports = generators.Base.extend({
 
   install: function () {
     this.installDependencies({ bower: false });
+
+    if (this.useAsciiDoc) {
+      try {
+        execSync('gem install bundler', { stdio: [0, 1, 2] });
+        execSync('bundle --path=.bundle/rubygems', { stdio: [0, 1, 2] });
```

Comment:
I'd like to change this path from .bundle/rubygems to .bundle/gems. My original recommendation was incorrect, so that's my fault. Sorry to change my mind on you.